### PR TITLE
MGMT-2223 - Add request ID on Auth failure

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

File: pkg/auth/auth_handler.go [hunk 1]
```diff
@@ -219,6 +220,7 @@ func (a *AuthHandler) CreateAuthenticator() func(name, in string, authenticate s
 			}
 			p, err := authenticate(token)
 			if err != nil {
+				a.log.Errorf("Fail to authenticate: Req_id %s, Error %v", requestid.FromContext(r.Context()), err)
```
Comment (filanov) [inline pkg/auth/auth_handler.go:223]: just to be in a standard suggesting to create a new logger as any other place:
```
log := logutil.FromContext(r.Context(), a.log)
```
Then you will not need to write the request id manually and work as any other place.

File: pkg/auth/auth_handler.go [hunk 2]
```diff
@@ -219,6 +220,8 @@ func (a *AuthHandler) CreateAuthenticator() func(name, in string, authenticate s
 			}
 			p, err := authenticate(token)
 			if err != nil {
+				log := logutil.FromContext(r.Context(), a.log)
```
Comment (filanov) [inline pkg/auth/auth_handler.go:223]: probably should move the beginning of the function then if we want to add some logs we will be able to do so.

File: pkg/auth/auth_handler.go (hunk 1 above)
Comment (rollandf) [reply pkg/auth/auth_handler.go]: Done

File: pkg/auth/auth_handler.go (hunk 2 above)
Comment (rollandf) [reply pkg/auth/auth_handler.go]: Done

**Review by filanov — APPROVED** [review]

**Pull request merged**